src/applications/vaos/reducers/newAppointment.js

```diff
@@ -172,7 +172,7 @@ export default function formReducer(state = initialState, action) {
         },
       };
     }
-    case FORM_RESET: {
+    case FORM_CLOSED_CONFIRMATION_PAGE: {
```

Comment:
Do we foresee any other scenarios where we would need to reset the form aside from closing the confirmation page? e.g. what happens if the user abandons the flow by navigating somewhere else on va.gov and returns to vaos? I think the state would persist, right?

Reply:
I don't think we'd have any other scenarios, but even if we do, I think we should try to keep actions more aligned with things the user is doing, rather than things that describe the internal workings of our code (i.e. the user isn't trying to reset the form here, they're leaving the page).

If you're in the middle of the form and you click the breadcrumb to go back to the list page, we will keep your state. I think that's ok. However, if you go anywhere outside of the VAOS app on va.gov, you'll lose your form state, because there will be a new page load. We should warn a user when this is going to happen with a beforeunload message, but we don't do that yet.

Reply:
Okay, makes sense!